src/Magento/FunctionalTestingFramework/Suite/Util/SuiteObjectExtractor.php

```diff
@@ -63,36 +63,75 @@ public function parseSuiteDataIntoObjects($parsedSuiteData)
             return $suiteObjects;
         }
 
+        $noError = true;
+        $suiteSkipped = 0;
         foreach ($parsedSuiteData[self::SUITE_ROOT_TAG] as $parsedSuite) {
             if (!is_array($parsedSuite)) {
                 // skip non array items parsed from suite (suite objects will always be arrays)
                 continue;
             }
 
-            $this->validateSuiteName($parsedSuite);
+            try {
+                $this->validateSuiteName($parsedSuite);
+
+                // extract include and exclude references
+                $groupTestsToInclude = $parsedSuite[self::INCLUDE_TAG_NAME] ?? [];
+                $groupTestsToExclude = $parsedSuite[self::EXCLUDE_TAG_NAME] ?? [];
 
-            //extract include and exclude references
-            $groupTestsToInclude = $parsedSuite[self::INCLUDE_TAG_NAME] ?? [];
-            $groupTestsToExclude = $parsedSuite[self::EXCLUDE_TAG_NAME] ?? [];
+                // resolve references as test objects
+                // continue if failed in include
+                $include = $this->extractTestObjectsFromSuiteRef($groupTestsToInclude);
+                $includeTests = $include['objects'] ?? [];
+                $stepError = $include['status'] ?? 0;
+                $includeMessage = '';
+                if ($stepError != 0) {
+                    $noError = false;
+                    $includeMessage = strval($stepError) . " test(s) not included for suite \""
```

Comment:
I see this error when running generate:tests though i see the test generated. 
`1 test(s) not included for suite "AppConfigDumpSuite"`

Reply:
Can you check again? Please add command used to see the error.

Reply:
1. I introduced a schema error in `AdminLoginSuccessfulTest` for this step -
`<actionGroup1 ref="AssertAdminSuccessLoginActionGroup" stepKey="assertLoggedIn"/>`

2. then I ran `bin/mftf generate:tests`

this is the output I get:
```
FAILURE: Schema validation error Element 'actionGroup1': This element is not expected.Line: 25
Unable to parse suite "SecureStorefrontURLSuite"NOTICE: 164 Section name violations detected. See mftf.log for details.[]
NOTICE: 617 section element name violations detected. See mftf.log for details.[]
NOTICE: 301 data entity name violations detected. See mftf.log for details.[]
NOTICE: 3 data entity key violations detected. See mftf.log for details.[]
1 test(s) not included for suite "AppConfigDumpSuite"NOTICE: 10 action group name violations detected. See mftf.log for details.[]
NOTICE: 35 Page name violations detected. See mftf.log for details.[]
ERROR: 2 Suite failed to generate. See mftf.log for details.NOTICE: Suite 'AppConfigDumpSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'WYSIWYGDisabledSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'MagentoProductionModeOnlyTestSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'MagentoDeveloperModeOnlyTestSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'SearchEngineElasticsearchSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'InContextPaypalSuite' contains no tests and won't be generated.
```

Reply:
1. I introduced a schema error in `AdminLoginSuccessfulTest` for this step -
`<actionGroup1 ref="AssertAdminSuccessLoginActionGroup" stepKey="assertLoggedIn"/>`

2. then I ran `bin/mftf generate:tests`

this is the output I get:
```
FAILURE: Schema validation error Element 'actionGroup1': This element is not expected.Line: 25
Unable to parse suite "SecureStorefrontURLSuite"NOTICE: 164 Section name violations detected. See mftf.log for details.[]
NOTICE: 617 section element name violations detected. See mftf.log for details.[]
NOTICE: 301 data entity name violations detected. See mftf.log for details.[]
NOTICE: 3 data entity key violations detected. See mftf.log for details.[]
1 test(s) not included for suite "AppConfigDumpSuite"NOTICE: 10 action group name violations detected. See mftf.log for details.[]
NOTICE: 35 Page name violations detected. See mftf.log for details.[]
ERROR: 2 Suite failed to generate. See mftf.log for details.NOTICE: Suite 'AppConfigDumpSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'WYSIWYGDisabledSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'MagentoProductionModeOnlyTestSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'MagentoDeveloperModeOnlyTestSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'SearchEngineElasticsearchSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'InContextPaypalSuite' contains no tests and won't be generated.
```

Reply:
Fixed